sql/moz-fx-data-shared-prod/fenix_derived/clients_city_seen_v1/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+friendly_name: fenix Clients City Seen
+description: |-
+owners:
+  - wichan@mozilla.com
+  - rbaffourawuah@mozilla.com
+labels:
+  change_controlled: true
```

Comment:
This label requires [adding the table to the CODEOWNERS](https://mozilla.github.io/bigquery-etl/cookbooks/common_workflows/#setting-up-change-control-to-code-files) file with an owner -preferably a GitHub identity-.